LemonUI.Example/Demo.cs

```diff
@@ -86,6 +90,7 @@ public static class Basics
         private static readonly NativeCheckboxItem useMouse = new NativeCheckboxItem("Use Mouse", "If the mouse should be used for selecting different items.", menu.UseMouse);
         private static readonly NativeItem flip = new NativeItem("Flip", "Flips the Menu from the left to the Right.");
         private static readonly NativeItem showHack = new NativeItem("Show Hack", "Shows the hacking minigame.");
+        private static readonly NativeItem randomFont = new NativeItem("RandomFont", "Random font to all text.");
```

Comment:
Maybe a better Title and Description that explain stuff more clearly?

```suggestion
        private static readonly NativeItem randomFont = new NativeItem("Set a Random Font", "Sets a random font for all of the Elements in the menu.");
```